packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,126 @@
+# Calendar
+
+## Component Description
+
+The Calendar lets users view and select a single date or a range of dates. The calendar has five different views: date, month, week, year, and range-picker.
+
+### Calendar vs. Date Picker Implementation
+
+The calendar and the date picker have significant overlap in inputs, outputs, and events. The difference is that the date picker is essentially a calendar combined with an input field, in which the input field can can interact with the calendar by opening and closing the calendar and displaying/modifying the selected dates of the calendar. It is important to note that the calendar is the component that stores the selected dates, which is then exposed to the date picker.
+
+## Design Spec
+
+Due to the incomplete nature of the v9 calendar design spec, work on the component will be based on the calendar from the v8 api while using the skin of the calendar in the v9 date picker design spec.
+
+[Link to v9 Date Picker Design in Figma](https://www.figma.com/file/9EHmS5y1Rr7DCh7KgJBwuz/DatePicker?type=design&mode=design&t=E6sL2ffuuZfwCSRp-0)
+
+[Link to Fluent v8 Calendar Component](https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar)
+
+## Engineering Spec
+
+### Inputs
+
+| attribute                  | type                                                    | default           | description                                                                                                                       |
+| -------------------------- | ------------------------------------------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `type`                     | "date" \| "month" \| "week" \| "year" \| "range-picker" | "date"            | The type of the calendar                                                                                                          |
```

Comment:
Is this union type available from fluent? If not, create a standalone type from this union.